docs/en/solutions/acp/ACP_Solution_for_Using_Red_Hat_UBI_Images.md

```
@@ -0,0 +1,41 @@
+---
+id: KBxxx
+products:
+  - Alauda Container Platform
+kind:
+  - Solution
+---
+
+# ACP Solution for Using Red Hat UBI Images
+
+**NOTE**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)